tox.ini

```diff
@@ -1,18 +1,19 @@
 [tox]
-envlist = py27,pypy,flake8,py36-mypy
+#envlist = py27,pypy,flake8,py36-mypy
+envlist = pypy,flake8
```

Comment:
!

Reply:
crap. sorry, that's because I was trying to force test against pypy.